doc/classes/ArrayMesh.xml

```diff
@@ -7,19 +7,40 @@
 		The [ArrayMesh] is used to construct a [Mesh] by specifying the attributes as arrays.
 		The most basic example is the creation of a single triangle:
 		[codeblock]
+		[gdscript]
 		var vertices = PackedVector3Array()
 		vertices.push_back(Vector3(0, 1, 0))
 		vertices.push_back(Vector3(1, 0, 0))
 		vertices.push_back(Vector3(0, 0, 1))
-		# Initialize the ArrayMesh.
+
+		# Initialize the ArrayMesh
 		var arr_mesh = ArrayMesh.new()
 		var arrays = []
 		arrays.resize(ArrayMesh.ARRAY_MAX)
 		arrays[ArrayMesh.ARRAY_VERTEX] = vertices
-		# Create the Mesh.
+
+		# Create the Mesh
 		arr_mesh.add_surface_from_arrays(Mesh.PRIMITIVE_TRIANGLES, arrays)
 		var m = MeshInstance3D.new()
 		m.mesh = arr_mesh
+		[/gdscript]
+		[csharp]
+		var verticies = new Godot.Collections.Array&lt;Vector3&gt;();
```

Comment:
Some problems with a character here

Reply:
that's on purpose, because we have invalid XML otherwise (< and >)

Reply:
Ah! So it appears correctly when building the docs?

Reply:
yes